articles/faq/stay-on-plan-cost-more.md

```diff
@@ -1,19 +1,25 @@
 ---
 title: "If I stay on my current plan, will my monthly cost stay the same?"
-description: "Your monthly cost will remain the same for the lifetime of your account. However, your overages will be capped at your maximum sending from the last 90 days. If you intend to send more than your limit, we’d encourage you to take advantage of the discounts, and upgrade your account."
+description: "Your monthly cost will remain the same for the lifetime of your account. However, your overages will be capped at your maximum sending from the last 90 days plus 30%. If you intend to send more than your limit, we’d encourage you to take advantage of the discounts, and upgrade your account."
 ---
 
-Your monthly cost will remain the same for the lifetime of your account. However, your overages will be capped at your maximum sending from the last 90 days or the published plan volume, whichever is the greater. If you intend to send more than your limit, we’d encourage you to take advantage of the discounts, and upgrade your account.
+Your monthly cost will remain the same for the lifetime of your account. However, your overages will be capped at your maximum sending from the last 90 days or the published plan volume plus 30%, whichever is the greater. If you intend to send more than your limit, we’d encourage you to take advantage of the discounts, and upgrade your account.
 
-For example if you are on a 1M plan and have the following sending pattern for the last 90 days:
+
+**Example 1 - No overage**
```

Comment:
I suggest we use headers instead of bold here and below:

```
### Example 1 - No Overage
```

The more hashes you add, the smaller the header, up to a maximum of 6.